joint_teapot/workers/gitea.py

```diff
@@ -426,6 +431,115 @@ def unsubscribe_from_repos(self, pattern: str) -> None:
             )
             logger.info(f"Unsubscribed from {sub.name}")
 
+    def JOJ3_scoreboard(
```

Comment:
It should be separated functions. It is about coupling and cohesion.

Create a `joint_teapot/utils/joj3.py`.

`def generate_scoreboard(score_file_path: str, scoreboard_file_path: str) -> str` output string as the content of CSV file.

In the future, we will have `def generate_comment(score_file_path: str) -> str`.

Add a new function in `joint_teapot/workers/git.py` to commit and push to the repo.

Finally, in `joint_teapot/app.py`

1. call `self.git.repo_clean_and_checkout` to checkout the latest scoreboard repo
2. call that `generate_scoreboard` to get the content of CSV
3. write that content to the file
4. call your new function in `joint_teapot/workers/git.py` to commit and push to the repo